packages/ui-drawer-layout/src/DrawerLayout/DrawerContent/index.tsx

```diff
@@ -73,12 +73,11 @@ class DrawerContent extends Component<Props> {
   }
 
   static defaultProps = {
-    children: null,
+    role: 'region',
     // @ts-expect-error ts-migrate(6133) FIXME: 'el' is declared but its value is never read.
-    contentRef: (el: any) => {},
+    contentRef: (el: any) => {}, // TODO if this is set to undefined _content is never set
     // @ts-expect-error ts-migrate(6133) FIXME: 'node' is declared but its value is never read.
-    onSizeChange: (node: any) => {},
-    role: 'region'
+    onSizeChange: (node: any) => {} // TODO this should not need a default value
```

Comment:
this was a tricky one, the component breaks when these are set to null/undefined. We should fix this in the future.